### PR TITLE
[TACHYON-921] Renamed User to Session across the codebase

### PR DESCRIPTION
**Pull request by yupeng9**
https://tachyon.atlassian.net/browse/TACHYON-921

Renamed User to Session across the codebase

### CODE REVIEW

File: clients/unshaded/src/main/java/tachyon/client/TachyonFS.java [hunk 1]
```diff
@@ -705,13 +705,13 @@ public synchronized TachyonURI getUri() {
   }
 
   /**
-   * Returns the userId of the worker client. This is only used for testing.
+   * Returns the sessionId of the worker client. This is only used for testing.
```
Comment (gpang) [inline clients/unshaded/src/main/java/tachyon/client/TachyonFS.java]: you can probably just use the @return javadoc.

File: common/src/main/java/tachyon/Constants.java [hunk 2]
```diff
@@ -159,7 +159,7 @@
   public static final String WORKER_HEARTBEAT_TIMEOUT_MS = "tachyon.worker.heartbeat.timeout.ms";
   public static final String WORKER_TO_MASTER_HEARTBEAT_INTERVAL_MS =
       "tachyon.worker.to.master.heartbeat.interval.ms";
-  public static final String WORKER_USER_TIMEOUT_MS = "tachyon.worker.user.timeout.ms";
+  public static final String WORKER_SESSION_TIMEOUT_MS = "tachyon.worker.session.timeout.ms";
```
Comment (gpang) [inline common/src/main/java/tachyon/Constants.java]: this should be updated in the documentation website.

**Comment by gpang** [comment]
I left a few comments. Thanks for the PR!

File: common/src/main/java/tachyon/Constants.java (hunk 2 above)
Comment (yupeng9) [inline common/src/main/java/tachyon/Constants.java]: Thanks for pointing it out. Updated

File: clients/unshaded/src/main/java/tachyon/client/TachyonFS.java [hunk 3]
```diff
@@ -169,7 +169,7 @@ public static synchronized TachyonFS get(TachyonConf tachyonConf) {
   private final Map<Long, Set<Integer>> mLockedBlockIds = new HashMap<Long, Set<Integer>>();
   /** Mapping from block id to path of the block locked */
   private final Map<Long, String> mLockedBlockIdToPath = new HashMap<Long, String>();
-  /** Each user facing block has a unique block lock id. */
+  /** Each session facing block has a unique block lock id. */
```
Comment (calvinjia) [inline clients/unshaded/src/main/java/tachyon/client/TachyonFS.java]: This sounds kind of weird.

File: integration-tests/src/test/java/tachyon/worker/BlockServiceHandlerIntegrationTest.java [hunk 4]
```diff
@@ -103,7 +103,7 @@ public void addCheckpointTest() throws Exception {
     TachyonFile file = mTfs.open(new TachyonURI("/testFile"));
     final int blockSize = (int) WORKER_CAPACITY_BYTES / 10;
 
-    String tmpFolder = mWorkerServiceHandler.getUserUfsTempFolder(USER_ID);
+    String tmpFolder = mWorkerServiceHandler.getSessionUfsTempFolder(USER_ID);
```
Comment (apc999) [inline integration-tests/src/test/java/tachyon/worker/BlockServiceHandlerIntegrationTest.java]: USER_ID -> SESSION_ID

**Comment by apc999** [comment]
Thanks for the patch! looks good to me almost. We can merge this PR after two minor comments addressed.

File: docs/Configuration-Settings.md [hunk 5]
```diff
@@ -385,9 +385,9 @@ number.
   <td>Kerberos principal for Tachyon worker.</td>
 </tr>
 <tr>
-  <td>tachyon.worker.user.timeout.ms</td>
+  <td>tachyon.worker.session.timeout.ms</td>
   <td>10000</td>
-  <td>Timeout between worker and user connection indicating a lost user connection.  Specified in milliseconds</td>
+  <td>Timeout between worker and clent connection indicating a lost session connection.  Specified in milliseconds</td>
```
Comment (calvinjia) [inline docs/Configuration-Settings.md]: client

**Comment by yupeng9** [comment]
Thanks for the review. Addressed all the comments.

**Comment by apc999** [comment]
+1 LGTM. Thanks!

**Comment by gpang** [comment]
LGTM

**Pull request merged**